exercises/hello-world/hello-world_example.d

```diff
@@ -0,0 +1,7 @@
+module helloworld;
```

Comment:
What is the dlang convention for filenames? hyphens or underscores? I think we should choose one or the other and be consistent.